Unit 10
595
7795
18595
17995
23995
10795
20395
5995
11395
16195
26395
23395
0
3595
14995
19195
1195
5395
14395
29395
2395
12595
11995
22195
28795
2995
13195
8995
17395
26995
20995
9595
19795
21595
10195
22795
1795
15595
16795
24595
4195
7195
28195
8395
27595
6595
13795
25195
4795
25795
Unit 02
23395
6595
19795
20395
36595
11995
2395
13195
595
25795
15595
28795
29995
37195
5395
22195
35995
0
17395
4195
11395
23995
14995
2995
31195
32395
29395
4795
16195
33595
8395
13795
32995
26395
9595
21595
12595
37795
10795
14395
30595
26995
25195
27595
35395
38395
7195
22795
1795
10195
19195
18595
31795
24595
5995
16795
20995
17995
34195
3595
8995
34795
38995
28195
7795
1195
multiprocessing.pool.RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/home/njw275/miniconda3/envs/birds/lib/python3.6/site-packages/joblib/_parallel_backends.py", line 350, in __call__
    return self.func(*args, **kwargs)
  File "/home/njw275/miniconda3/envs/birds/lib/python3.6/site-packages/joblib/parallel.py", line 131, in __call__
    return [func(*args, **kwargs) for func, args, kwargs in self.items]
  File "/home/njw275/miniconda3/envs/birds/lib/python3.6/site-packages/joblib/parallel.py", line 131, in <listcomp>
    return [func(*args, **kwargs) for func, args, kwargs in self.items]
  File "medianfilt_onsetsAndPeaks.py", line 32, in createOnsetsAndPeaks
    b = sci.medfilt(b, kernel_size=5)
NameError: name 'sci' is not defined

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/njw275/miniconda3/envs/birds/lib/python3.6/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/njw275/miniconda3/envs/birds/lib/python3.6/site-packages/joblib/_parallel_backends.py", line 359, in __call__
    raise TransportableException(text, e_type)
joblib.my_exceptions.TransportableException: TransportableException
___________________________________________________________________________
NameError                                          Wed Jul 12 17:05:29 2017
PID: 50051      Python 3.6.1: /home/njw275/miniconda3/envs/birds/bin/python
...........................................................................
/home/njw275/miniconda3/envs/birds/lib/python3.6/site-packages/joblib/parallel.py in __call__(self=<joblib.parallel.BatchedCalls object>)
    126     def __init__(self, iterator_slice):
    127         self.items = list(iterator_slice)
    128         self._size = len(self.items)
    129 
    130     def __call__(self):
--> 131         return [func(*args, **kwargs) for func, args, kwargs in self.items]
        self.items = [(<function createOnsetsAndPeaks>, (1,), {})]
    132 
    133     def __len__(self):
    134         return self._size
    135 

...........................................................................
/home/njw275/miniconda3/envs/birds/lib/python3.6/site-packages/joblib/parallel.py in <listcomp>(.0=<list_iterator object>)
    126     def __init__(self, iterator_slice):
    127         self.items = list(iterator_slice)
    128         self._size = len(self.items)
    129 
    130     def __call__(self):
--> 131         return [func(*args, **kwargs) for func, args, kwargs in self.items]
        func = <function createOnsetsAndPeaks>
        args = (1,)
        kwargs = {}
    132 
    133     def __len__(self):
    134         return self._size
    135 

...........................................................................
/home/njw275/NJW-BirdVox/src/python_files/medianfilt_onsetsAndPeaks.py in createOnsetsAndPeaks(unit=1)
     27         print(start_time)
     28         #det.onset(unit, start_time, fmin_input, h_length,feature)
     29         # Peak picks (read from file above, write to a new file)
     30     for start_time in start_times:
     31         a = p.get_detections(unit)
---> 32         b = sci.medfilt(b, kernel_size=5)
        b = undefined
     33         saveFile = get_filtered_detections(unit)
     34         np.save(saveFile, b)
     35         det.peak_picking(unit, start_time, h_duration)
     36 

NameError: name 'sci' is not defined
___________________________________________________________________________
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/njw275/miniconda3/envs/birds/lib/python3.6/site-packages/joblib/parallel.py", line 699, in retrieve
    self._output.extend(job.get(timeout=self.timeout))
  File "/home/njw275/miniconda3/envs/birds/lib/python3.6/multiprocessing/pool.py", line 608, in get
    raise self._value
joblib.my_exceptions.TransportableException: TransportableException
___________________________________________________________________________
NameError                                          Wed Jul 12 17:05:29 2017
PID: 50051      Python 3.6.1: /home/njw275/miniconda3/envs/birds/bin/python
...........................................................................
/home/njw275/miniconda3/envs/birds/lib/python3.6/site-packages/joblib/parallel.py in __call__(self=<joblib.parallel.BatchedCalls object>)
    126     def __init__(self, iterator_slice):
    127         self.items = list(iterator_slice)
    128         self._size = len(self.items)
    129 
    130     def __call__(self):
--> 131         return [func(*args, **kwargs) for func, args, kwargs in self.items]
        self.items = [(<function createOnsetsAndPeaks>, (1,), {})]
    132 
    133     def __len__(self):
    134         return self._size
    135 

...........................................................................
/home/njw275/miniconda3/envs/birds/lib/python3.6/site-packages/joblib/parallel.py in <listcomp>(.0=<list_iterator object>)
    126     def __init__(self, iterator_slice):
    127         self.items = list(iterator_slice)
    128         self._size = len(self.items)
    129 
    130     def __call__(self):
--> 131         return [func(*args, **kwargs) for func, args, kwargs in self.items]
        func = <function createOnsetsAndPeaks>
        args = (1,)
        kwargs = {}
    132 
    133     def __len__(self):
    134         return self._size
    135 

...........................................................................
/home/njw275/NJW-BirdVox/src/python_files/medianfilt_onsetsAndPeaks.py in createOnsetsAndPeaks(unit=1)
     27         print(start_time)
     28         #det.onset(unit, start_time, fmin_input, h_length,feature)
     29         # Peak picks (read from file above, write to a new file)
     30     for start_time in start_times:
     31         a = p.get_detections(unit)
---> 32         b = sci.medfilt(b, kernel_size=5)
        b = undefined
     33         saveFile = get_filtered_detections(unit)
     34         np.save(saveFile, b)
     35         det.peak_picking(unit, start_time, h_duration)
     36 

NameError: name 'sci' is not defined
___________________________________________________________________________

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "medianfilt_onsetsAndPeaks.py", line 37, in <module>
    Parallel(n_jobs=-1)(delayed(createOnsetsAndPeaks)(unit) for unit in units)
  File "/home/njw275/miniconda3/envs/birds/lib/python3.6/site-packages/joblib/parallel.py", line 789, in __call__
    self.retrieve()
  File "/home/njw275/miniconda3/envs/birds/lib/python3.6/site-packages/joblib/parallel.py", line 740, in retrieve
    raise exception
joblib.my_exceptions.JoblibNameError: JoblibNameError
___________________________________________________________________________
Multiprocessing exception:
...........................................................................
/home/njw275/NJW-BirdVox/src/python_files/medianfilt_onsetsAndPeaks.py in <module>()
     32         b = sci.medfilt(b, kernel_size=5)
     33         saveFile = get_filtered_detections(unit)
     34         np.save(saveFile, b)
     35         det.peak_picking(unit, start_time, h_duration)
     36 
---> 37 Parallel(n_jobs=-1)(delayed(createOnsetsAndPeaks)(unit) for unit in units)

...........................................................................
/home/njw275/miniconda3/envs/birds/lib/python3.6/site-packages/joblib/parallel.py in __call__(self=Parallel(n_jobs=-1), iterable=<generator object <genexpr>>)
    784             if pre_dispatch == "all" or n_jobs == 1:
    785                 # The iterable was consumed all at once by the above for loop.
    786                 # No need to wait for async callbacks to trigger to
    787                 # consumption.
    788                 self._iterating = False
--> 789             self.retrieve()
        self.retrieve = <bound method Parallel.retrieve of Parallel(n_jobs=-1)>
    790             # Make sure that we get a last message telling us we are done
    791             elapsed_time = time.time() - self._start_time
    792             self._print('Done %3i out of %3i | elapsed: %s finished',
    793                         (len(self._output), len(self._output),

---------------------------------------------------------------------------
Sub-process traceback:
---------------------------------------------------------------------------
NameError                                          Wed Jul 12 17:05:29 2017
PID: 50051      Python 3.6.1: /home/njw275/miniconda3/envs/birds/bin/python
...........................................................................
/home/njw275/miniconda3/envs/birds/lib/python3.6/site-packages/joblib/parallel.py in __call__(self=<joblib.parallel.BatchedCalls object>)
    126     def __init__(self, iterator_slice):
    127         self.items = list(iterator_slice)
    128         self._size = len(self.items)
    129 
    130     def __call__(self):
--> 131         return [func(*args, **kwargs) for func, args, kwargs in self.items]
        self.items = [(<function createOnsetsAndPeaks>, (1,), {})]
    132 
    133     def __len__(self):
    134         return self._size
    135 

...........................................................................
/home/njw275/miniconda3/envs/birds/lib/python3.6/site-packages/joblib/parallel.py in <listcomp>(.0=<list_iterator object>)
    126     def __init__(self, iterator_slice):
    127         self.items = list(iterator_slice)
    128         self._size = len(self.items)
    129 
    130     def __call__(self):
--> 131         return [func(*args, **kwargs) for func, args, kwargs in self.items]
        func = <function createOnsetsAndPeaks>
        args = (1,)
        kwargs = {}
    132 
    133     def __len__(self):
    134         return self._size
    135 

...........................................................................
/home/njw275/NJW-BirdVox/src/python_files/medianfilt_onsetsAndPeaks.py in createOnsetsAndPeaks(unit=1)
     27         print(start_time)
     28         #det.onset(unit, start_time, fmin_input, h_length,feature)
     29         # Peak picks (read from file above, write to a new file)
     30     for start_time in start_times:
     31         a = p.get_detections(unit)
---> 32         b = sci.medfilt(b, kernel_size=5)
        b = undefined
     33         saveFile = get_filtered_detections(unit)
     34         np.save(saveFile, b)
     35         det.peak_picking(unit, start_time, h_duration)
     36 

NameError: name 'sci' is not defined
___________________________________________________________________________
